axelrod/strategies/titfortat.py

```diff
@@ -92,6 +93,41 @@ class TwoTitsForTat(Player):
     def strategy(opponent: Player) -> Action:
         return D if D in opponent.history[-2:] else C
 
+class DynamicTwoTitsForTat(Player):
+    """
+    A player starts by cooperating and then punishes its opponent's 
+    defectiions by opponent with a dynamic bias based off of the 
+    opponents ratio of cooperations to total moves (so their current 
+    probability of cooperating towards cooporating regardless of the 
+    move (aka: forgiveness)).
```

Comment:
"(so their current probability of cooperating towards cooporating regardless of the move (aka: forgiveness))."

Sorry, but I don't understand what that line means.